### DISCUSSION TITLE
Isar 4 prerelease affects app size

### DISCUSSION THREAD

**Discussion by kxnstandin** [isar/isar#1447] [Help]
I started a new project in flutter and when I build a release apk with Isar 3 stable, the app size is about 8-9MB.
When I build the same app with Isar 4 dev-14 prerelease, the app size jumps up to 20+MB.

Is this normal for the prerelease version and will be fixed with Isar 4 stable (which hopefully releases soon)?

Also, would you already start production code with Isar 4 until the stable version comes out?

**Comment by Farg0k** [comment]
Why has the size of applications increased so much since v4?

**Comment by Cierra-Runis** [comment]
```gradle
 /// TIPS: https://juejin.cn/post/7330471268033478666
    packagingOptions {
        dex {
            useLegacyPackaging true
        }
        jniLibs {
            useLegacyPackaging true
        }
    }
```

**Reply by Farg0k** [reply]
thx, its work
